src/api-calls.js

```diff
@@ -0,0 +1,48 @@
+function fetchData(file) {
+  return fetch(`http://localhost:3001/api/v1/${file}`).then((response) =>
+    response.json()
+  );
+}
+
+function postData(file) {
```

Comment:
fetchData file will probably need some refactoring.  Will meet with mentor to get a better understanding.